src/main/java/seedu/address/model/profile/exceptions/SimilarProfileException.java

```diff
@@ -4,8 +4,8 @@
  * Signals that the operation will result in duplicate Profiles (Profiles are duplicates if they have the same
  * identity).
  */
-public class DuplicateProfileException extends RuntimeException {
-    public DuplicateProfileException() {
+public class SimilarProfileException extends RuntimeException {
+    public SimilarProfileException() {
         super("Operation would result in duplicate profiles");
```

Comment:
Perhaps the message here can be updated to reflect the renaming of the exception, from `duplicate profiles` to `similar profiles`.